clients/imodels-client-management/src/base/internal/OperationsBase.ts

```diff
@@ -128,7 +128,7 @@ export class OperationsBase<TOptions extends OperationsBaseOptions> {
     if (params.preferReturn)
       headers[Constants.headers.prefer] = `return=${params.preferReturn}`;
 
-    if (params.contentType)
+    if (params.contentType && params.body !== undefined && JSON.stringify(params.body) !== "{}")
```

Comment:
`{}` logically may be empty body, but in terms forming an HTTP request it is not. This could be successfully deserialized into an object with all its optional properties set to undefined. Remove this clause.